src/job-exporter/deploy/job-exporter.yaml.template

```diff
@@ -41,6 +41,7 @@ spec:
             port: {{ cluster_cfg["job-exporter"]["port"] }}
           initialDelaySeconds: 3
           periodSeconds: 30
+          timeoutSeconds: 3
```

Comment:
Does this 3 a little small ? which maybe easy to see readiness probe failed ?

Reply:
The default is 1s. Our bed shows 1s won't be too small.

Reply:
@xudifsd , please consider setting the value to the worst case.

Reply:
maybe 10?